Homework-june-13/Tamara_Mkrtchyan/README.md

```diff
@@ -0,0 +1,7 @@
+## creating and deleting instances with all necessary resources
+
+### run
+```
+./aws_ec2.sh --create <name>
+./aws_ec2.sh --delete <name>
+```
```

Comment:
```suggestion
```
usage
./aws_ec2.sh [--option] [name]
options:
    --create  Creating ec2 instance with all necessary resources
    --delete  Deleting ec2 instance and all resources that created by --create option
name:
    any string that is acceptable for AWS resources Name tag

```
```